src/TraceEvent/TraceEventSession.cs

```diff
@@ -2391,6 +2453,15 @@ private static unsafe void CopyStringToPtr(char* toPtr, string str)
             }
         }
 
+        /// <summary>
+        /// Get the max number of last branch recording sources that can be specified at the same time.
+        /// </summary>
+        public static int GetMaxLastBranchRecordingSources()
+        {
+            const int ETW_MAX_LBR_EVENTS = 4;
```

Comment:
Is there a way to determine this from the OS that we should be using instead of a constant?

Reply:
This is defined in the Windows SDK headers, I don't think there is an OS API to get it. But perhaps perfview has a native component I expose this via to avoid having to hardcode it like this?

Reply:
Not easily - this is OK.  Thank you.